plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/CredentialHelperTest.java

```diff
@@ -43,7 +43,8 @@ public void testToJsonFile() throws IOException {
     Path jsonFile = tempFolder.getRoot().toPath().resolve("credential-for-gcloud.json");
     CredentialHelper.toJsonFile(credential, jsonFile);
 
-    try (InputStream in = new FileInputStream(jsonFile.toFile());
+    try (InputStream in = Files.newInputStream(jsonFile);
+        // todo needs character set
```

Comment:
You can use `Files.newBufferedReader()` and give a charset, and the code will be further simplified as well.